hikari/api/rest.py

```diff
@@ -2260,7 +2888,30 @@ async def edit_my_nick(
         *,
         reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> None:
-        ...
+        """Edit the associated token's member nick.
```

Comment:
Something along the lines of "Edit the associated token's nickname in the provided guild." might be a bit easier to read.